### ISSUE TITLE
I'm hitting the line limit

### ISSUE THREAD

**Issue by JoeCotellese** [jaesung-0o0/ChatUI#20]
Hi, I'm hitting the line limit for results returned by open ai. I see that it's hard-coded to 10. Is there an easy way to make this configurable?

**Comment by jaesung-0o0** [comment]
Hi, @JoeCotellese . Thanks for letting me know your issue that greatly contributes to the development of ChatUI.

### 1.0.0-beta.6

🚀 I released **1.0.0-beta.6** to support line limit in `MessageRow` and character limit in `MessageField`. Now both have an unlimited line(or character) as a default.

Here is an example usage:

```swift
MessageList(messages) { message
    // rowContent
    MessageRow(message: message, lineLimit: 10) // 🌟 set `lineLimit`. The default is `nil` which means unlimited
        .padding(.top, 12)
}

MessageField(characterLimit: 300) { messageStyle in // 🌟 set `characterLimit`. The default is `nil` which means unlimited
    // send message...
}
```

### Additional Information

💡 It is not necessary to use `MessageRow` in the `rowContent` of `MessageList`. You can design your own message row that suits your app's needs by using the **`messageStyle(_:)` modifier in `View`** that is provided by ChatUI.

- Example usage(ChatGPT-like):

```swift
MessageList(messages) { message in
    HStack(alignment: .top, spacing: 8) {
        AsyncImage(url: message.sender.imageURL) { image in
            image
                .resizable()
                .messageStyle(.senderProfile)
            } placeholder: {
                Image(systemName: "person.crop.circle.fill")
                    .resizable()
                    .messageStyle(.senderProfile)
                    .foregroundColor(appearance.secondary)
            }

        switch style {
        case .text(let text):
            let markdown = LocalizedStringKey(text)
            Text(markdown)
                .fixedSize(horizontal: false, vertical: true) // or use `.lineLimit(nil`
                .tint(isMyMessage ? appearance.prominentLink : appearance.link)
                .messageStyle(isMyMessage ? .localBody({LINE.LIMIT}) : .remoteBody({LINE.LIMIT}))
    }
}
```

**Comment by jaesung-0o0** [comment]
@JoeCotellese
Great! I'll close this issue! 
If you have any issues regarding the line limit, please re-open or create a new issue.🌟
